### PR TITLE
[e2e] External sites - update /customize-store tests

### PR DESCRIPTION
**Pull request by Stojdza**
### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:

Updates to the current `e2e-pw` setup, so tests are passing against the Pressable environment as well. 

In this PR, we're covering tests in /customize-store folder.

### How to test the changes in this Pull Request:

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. CI green - to be sure that changes in this PR are not affecting wp-env execution.
2. Locally - checkout this branch and run `pnpm test:e2e:with-env default-pressable`. In `playwright.config.js` you can comment everything except this:
```
'**/customize-store/**/*.spec.js',
```
3. In the code, I put comments on why these changes.

### Changelog entry

-   [x] Automatically create a changelog entry from the details below.

-   [ ] This Pull Request does not require a changelog entry. (Comment required below)

<details>

<summary>Changelog Entry Details</summary>

#### Significance

-   [x] Patch
-   [ ] Minor
-   [ ] Major

#### Type

-   [ ] Fix - Fixes an existing bug
-   [ ] Add - Adds functionality
-   [x] Update - Update existing functionality
-   [ ] Dev - Development related task
-   [ ] Tweak - A minor adjustment to the codebase
-   [ ] Performance - Address performance issues
-   [ ] Enhancement - Improvement to existing functionality

#### Message 

Update /customize-store tests, so they are passing against Pressable env.

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>

### CODE REVIEW

File: plugins/woocommerce/tests/e2e-pw/tests/customize-store/assembler/color-picker.spec.js [hunk 1]
```diff
@@ -412,105 +412,105 @@ test.describe( 'Assembler -> Color Pickers', { tag: '@gutenberg' }, () => {
...
-				'.woocommerce-customize-store_global-styles-variations_item'
-			)
-			.last();
-
-		await colorPicker.click();
-
-		await assembler.locator( '[aria-label="Back"]' ).click();
+	test(
+		'Selected color palette should be applied on the frontend',
+		{ tag: '@skip-on-default-pressable' },
```
Comment (Stojdza) [inline plugins/woocommerce/tests/e2e-pw/tests/customize-store/assembler/color-picker.spec.js:417]: Number of introductory patterns not same on Pressable, 10 vs. 4.

`tag: '@skip-on-default-pressable'` is the only change. Everything else is ESLint styling update.

File: plugins/woocommerce/tests/e2e-pw/tests/customize-store/assembler/homepage.spec.js [hunk 2]
```diff
@@ -314,22 +314,23 @@ test.describe( 'Homepage tracking banner', () => {
...
-	} ) => {
-		await setOption(
-			request,
-			baseURL,
-			'woocommerce_allow_tracking',
-			'yes'
-		);
+	test(
+		'Should not show the "Want more patterns?" banner when tracking is allowed',
+		{ tag: '@skip-on-default-pressable' },
```
Comment (Stojdza) [inline plugins/woocommerce/tests/e2e-pw/tests/customize-store/assembler/homepage.spec.js:319]: This banner is always present on Pressable, with the following message:

Screenshot 2024-08-27 at 19 46 34

`tag: '@skip-on-default-pressable'` is the only change. Everything else is ESLint styling update.

File: plugins/woocommerce/tests/e2e-pw/tests/customize-store/assembler/logo-picker/logo-picker.spec.js [hunk 3]
```diff
@@ -235,9 +235,10 @@ test.describe( 'Assembler -> Logo Picker', { tag: '@gutenberg' }, () => {
 		// alternative way to verify new site icon on the site
 		// verifying site icon shown in the new tab is impossible in headless mode
 		const date = new Date();
+		const month = ( date.getMonth() + 1 ).toString().padStart( 2, '0' );
```
Comment (Stojdza) [inline plugins/woocommerce/tests/e2e-pw/tests/customize-store/assembler/logo-picker/logo-picker.spec.js:238]: Attempt to fix date formatting here. `date.getMonth()` was returning `7`, and the expected format and number is `08`, at least that's where I found the uploaded logo for Pressable. This change works for Pressable, and it didn't break wp-env. 

@veljkho, it seems that you had [a similar issue](https://github.com/woocommerce/woocommerce/pull/49451#issuecomment-2226155507).

File: plugins/woocommerce/tests/e2e-pw/tests/customize-store/intro.spec.js [hunk 4]
```diff
@@ -115,34 +115,38 @@ test.describe(
 			).toBeVisible();
 		} );
 
-		test( 'it shows the "non default block theme" banner when the theme is a block theme different than TT4', async ( {
-			page,
-		} ) => {
-			await activateTheme( 'twentytwentythree' );
+		test(
+			'it shows the "non default block theme" banner when the theme is a block theme different than TT4',
+			{ tag: '@skip-on-default-pressable' },
```
Comment (Stojdza) [inline plugins/woocommerce/tests/e2e-pw/tests/customize-store/intro.spec.js:120]: The theme is activated through CLI, which is an option that we don't have right now for Pressable. 

`tag: '@skip-on-default-pressable'` is the only change (in two places in this file). Everything else is ESLint styling update.

File: plugins/woocommerce/tests/e2e-pw/tests/customize-store/intro.spec.js [hunk 5]
```diff
@@ -115,34 +115,38 @@ test.describe(
...
+			}
+		);
 
-		test( 'clicking on "Go to the Customizer" with a classic theme should go to the customizer', async ( {
-			page,
-		} ) => {
-			await activateTheme( 'twentytwenty' );
+		test(
+			'clicking on "Go to the Customizer" with a classic theme should go to the customizer',
+			{ tag: '@skip-on-default-pressable' },
```
Comment (Stojdza) [inline plugins/woocommerce/tests/e2e-pw/tests/customize-store/intro.spec.js:137]: The theme is activated through CLI, which is an option that we don't have right now for Pressable.

**Review by adimoldovan — APPROVED** [review]
LGTM 👍🏼

**Pull request merged**